src/main/java/com/based/basedsurvey/controller/WebEditSurveyController.java

```diff
@@ -73,6 +73,15 @@ public String editName(@RequestParam String prompt, @RequestParam long id) {
         return "redirect:/survey/"+id +"/edit";
     }
 
+    @PostMapping("survey/openSurvey")
+    public String enableButtons(@RequestParam long id, @RequestParam boolean enable) {
+        var survey = getSurvey(id);
```

Comment:
i think we should have the controllers not change the database for editing a question or survey if that survey is closed

Reply:
I like that, will be a change for the final milestone as we are running out of stuff (we are too based.)